src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java

```diff
@@ -345,6 +345,44 @@ private void doSubscribe(RoutingContext routingContext, JsonObject bodyAsJson) {
         }
     }
 
+    public void doListSubscriptions(RoutingContext routingContext) {
+        this.listSubscriptions(listSubscriptions -> {
```

Comment:
naming `listSubscriptions` it seems it's the actual list of subscriptions while it's the asyncresult, maybe we can use the same naming as other places like `done`, or `listSubscriptionsResult` ... whatever you prefer :-)